tests/NCronJob.Tests/NCronJob.Tests.csproj

```diff
@@ -33,12 +33,15 @@
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="9.0.0-*" />
   </ItemGroup>
 
-    <ItemGroup>
-        <Using Include="Xunit" />
-    </ItemGroup>
+  <ItemGroup>
+    <Using Include="Xunit" />
+  </ItemGroup>
 
-    <ItemGroup>
-      <ProjectReference Include="..\..\src\NCronJob\NCronJob.csproj" />
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\NCronJob\NCronJob.csproj" />
+  </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can be removed and should be tackled with #125

Reply:
Fixed